### ISSUE TITLE
player.avatar(value : String) & player.ext(Value : String , textColor : int);

### ISSUE THREAD

**Issue by coadra** [haxball/haxball-issues#762]
It would be very useful to get & set avatar values for create better codes like, life system, setting the lifes of the players to make 8man auto host or something like that. 
in this case, the /avatar would be locked in the room so thy cant change it.
and maybe, add player name extension to show something like Rank, LvL, something like Mod or Creator, Admin, Vip.
Thanks for reading!

**Comment by coadra** [comment]
@basro

**Comment by 4tires** [comment]
I agree, although you can discard the avatars and count the number of lifes in the chat. But yes, I think the avatar system is important for the flow of the game.
